app/vmalert/group.go

```diff
@@ -59,6 +61,9 @@ func newGroup(cfg config.Group, defaultInterval time.Duration, labels map[string
 		finishedCh:  make(chan struct{}),
 		updateCh:    make(chan *Group),
 	}
+	// error format of auth token is filter at rule file Parse
+	token, _ := auth.NewToken(cfg.Tenant)
```

Comment:
my thinking is that you can pass already parsed auth token from config and avoid double parsing